book/src/nostr/06-nip06.md

```diff
@@ -1 +1,71 @@
-# NIP-06
+## NIP-06
+
+In accordance with the [Bitcoin Improvement Proposal 0039 (BIP-39)](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki) we can derive Nostr keys using seed phrases as a source of entropy. This is handled by the [FromMnemonic](https://docs.rs/nostr/latest/nostr/nips/nip06/trait.FromMnemonic.html) Trait and its associated methods.
+
+The default functionality is to generate a single key-pair at the derivation path `0`. However, it is also possible to perform more advanced derivations by incrementing the `account`, enabling generation of many sets of keys from a single seed. More granular derivation path control (over `account`, `type` and `index`) exists within the `from_mnemonic_advanced` method. 
+
+For more examples of key generation please refer back to the [Keys section](https://rust-nostr.org/nostr/03-keys.html) of this book.
+
+### Key derivation from mnemonic seed phrase (NIP-06)
+
+<custom-tabs category="lang">
+
+<div slot="title">Rust</div>
+<section>
+
+TODO
+
+</section>
+
+<div slot="title">Python</div>
+<section>
+
+Using the `from_mnemonic()` function in conjunction with the `Keys` class to derived a basic set of Nostr keys from a 24 word seed phrase.
+
+Note that this example uses the `Mnemonic` class from the [python-mnemonic](https://github.com/trezor/python-mnemonic) package (the reference implementation of BIP-39) to randomly generate example seed phrases.
+
+```python,ignore
+{{#include ../../snippets/nostr/python/src/nip05.py:keys-from-seed24}}
```

Comment:
The path to snippets file is wrong in all NIP-06 examples (it's set `nip05` instead of `nip06`)

Reply:
*face-palm*